src/_data/members/members/adiati98.js

```diff
@@ -7,24 +7,31 @@ module.exports = {
   //
   // name - if not defined here, will default to your Name on github if defined, if not, then your username
   // name: 'Your Name',
+  name: 'Ayu Adiati'
   //
   // mainUrl: the url your name links to on the members page. defaults to your github profile
   // mainUrl: 'https://virtualcoffee.io',
   //
+  mainUrl: 'https://adiati.com'
```

Comment:
I think we need a comma here!

Reply:
Oops!!!!
Going to fix it! 

I was in panic mode for a moment there when all tests failed, @danieltott  🙈